Exiled.Events/Patches/Fixes/Scp1853Fix.cs

```diff
@@ -0,0 +1,43 @@
+// -----------------------------------------------------------------------
+// <copyright file="Scp1853Fix.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Generic
+{
+    using System;
+
+    using CustomPlayerEffects;
+
+    using HarmonyLib;
+    #pragma warning disable SA1313
+
+    /// <summary>
+    /// Patch the <see cref="Scp1853.OnUpdate"/>.
+    /// Fix Spamming EnableEffect.
+    /// </summary>
+    [HarmonyPatch(typeof(Scp1853), nameof(Scp1853.OnUpdate))]
```

Comment:
If someone made it in Transpiler i deleted this Prefix and let the guys with the transpiler post